src/IntersectionObserver.svelte

```diff
@@ -16,22 +17,32 @@
   /** @type {null | Entry} */
   export let entry = null;
   export let intersecting = false;
+  /** @type {null | Observer} */
```

Comment:
```suggestion
  /** @type {null | IntersectionObserver} */
```

The TypeScript interface is [`IntersectionObserver` ](https://github.com/microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L9701).

Reply:
Oops, bad copy/paste. 👍